### ISSUE TITLE
Error in module Prelude (Foreign)

### ISSUE THREAD

**Issue by dtulig** [purescript/purescript-prelude#68]
I'm getting the following error when building with `pulp build --optimise`:

```
* Bundling JavaScript...
Error in module Prelude (Foreign):

  The module could not be parsed:
  lexical error @ line 217 and column 14

* ERROR: Subcommand terminated with exit code 1
```

My versions are:

```
> pulp --version
Pulp version 8.2.0-rc.3
psc version 0.7.6.1
```

This also happens with pulp 8.1.0. I'm using `purescript-prelude` 0.1.5 as I noticed #65 and thought that might fix it. I believe it is easily reproduce-able by cloning https://github.com/slamdata/purescript-halogen-template, upgrading to `purescript-prelude` 0.1.5, and running `pulp build --optimise`.

Line 217 in foreign.js is: `case "\b": return "'\\b'";`. I don't have a lot of context for this implementation but did find that changing that block out for the unicode codes allows it to build:

```
      case "\x08": return "'\\b'";
      case "\x0C": return "'\\f'";
      case "\x0A": return "'\\n'";
      case "\x0D": return "'\\r'";
      case "\x09": return "'\\t'";
      case "\x0B": return "'\\v'"
```

This is necessary in `showCharImpl` and `showStringImpl`. I can put together a PR but wanted to make sure that using the unicode escape sequences was the right way to solve this.

**Comment by garyb** [comment]
This will be fixed in the next PS release, the problem comes from the version of `language-javascript` we're depending on in 0.8.3: purescript/purescript#1986

Downgrading to psc 0.8.2 is also a workaround for now.

**Comment by nwolverson** [comment]
I think this should now be resolved with 0.8.4?

**Comment by garyb** [comment]
Indeed it is, thanks for the reminder.
